core/server/worker/src/main/java/alluxio/worker/block/allocator/Allocator.java

```diff
@@ -43,16 +43,8 @@ private Factory() {} // prevent instantiation
     public static Allocator create(BlockMetadataView view) {
       BlockMetadataView metadataView = Preconditions.checkNotNull(view, "view");
       return CommonUtils.createNewClassInstance(
-<<<<<<< HEAD
-          Configuration.<Allocator>getClass(PropertyKey.WORKER_ALLOCATOR_CLASS),
-          new Class[] {BlockMetadataManagerView.class}, new Object[] {managerView});
-||||||| parent of ec9f9ceb90... Reduce the information allocator need in createBlockMeta
-          ServerConfiguration.<Allocator>getClass(PropertyKey.WORKER_ALLOCATOR_CLASS),
-          new Class[] {BlockMetadataManagerView.class}, new Object[] {managerView});
-=======
-          ServerConfiguration.<Allocator>getClass(PropertyKey.WORKER_ALLOCATOR_CLASS),
+          Configuration.getClass(PropertyKey.WORKER_ALLOCATOR_CLASS),
```

Comment:
Removing the `<Allocator>` type?

Reply:
IDE prompted me to remove it, since it can be inferred. Do we have a convention to write it out?